### ISSUE TITLE
Better handling of multi-day and multi-shift events

### ISSUE THREAD

**Issue by iandees** [binx/hrc-events#40]
The events API that we're querying seems to expose multi-shift and multi-day events with individual results for each shift or day in the response, resulting in stuff like this:

![image](https://cloud.githubusercontent.com/assets/261584/17911052/d4a53578-6959-11e6-8ba4-6b71cdaa8528.png)

It'd be nice if we could collapse those into a single result on the left and a single pin on the map and then show "upcoming shifts" or "upcoming days" or something when viewing the details of the event.

Looking at the results from the search API I don't see anything immediately obvious to collapse on, but maybe start with address and see what it looks like?

**Comment by wboykinm** [comment]
@iandees This is partly resolved by @kielni's excellent date slider addition, but I'm noticing that there's basically a schema problem with the Cleveland example above. Some events are coming in over the API as fundamental duplicates when there are shifts involved. For example, [this payload](https://gist.github.com/wboykinm/effe21f79814fea7be8570fc146e12b1) contains 12 events for the same day at the Cleveland downtown office. 10 of them are scheduled for the **same start time**. All have unique `lookupId`s, so they're all unique events as far as the API is concerned.

I dunno, but this seems like an issue to raise with the campaign, and not one we can easily solve here.

**Comment by iandees** [comment]
Yep, I agree. It's quite easy to make duplicate events in BSD, so I'm not surprised this is showing up in their response.
